sdk/client/src/stream/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+pub(crate) mod streaming_client;
+pub(crate) mod traits;
+pub(crate) mod websocket_client;
+
+pub use self::{streaming_client::StreamingClient, websocket_client::WebsocketClient};
```

Comment:
confusing about StreamingClient and WebsocketClient.....
As you use stream to name most things current implemented as Websocket, I would think StreamingClient is the websocket client connecting to server.
I can understand if you mark WebsocketClient as lower level implementation, but not sure what is purpose to publish it, any user should direct use WebsocketClient? if so, how they decide which one to use?